notebooks/official/matching_engine/sdk_matching_engine_create_stack_overflow_embeddings.ipynb

```diff
@@ -24,13 +24,12 @@
       ]
```

Comment:
<p>They should do the VPC stuff before any of the rest of the notebook since the user has to start a Workbench instance inside the VPC to do index_endpoint.match(...)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/2335/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='58'/>